src/logic/collectibles/store/selectors/index.ts

```diff
@@ -9,22 +9,30 @@ import { safeActiveAssetsSelector } from 'src/routes/safe/store/selectors'
 export const nftAssetsSelector = (state: AppReduxState): NFTAssets => state[NFT_ASSETS_REDUCER_ID]
 export const nftTokensSelector = (state: AppReduxState): NFTTokens => state[NFT_TOKENS_REDUCER_ID]
 
-export const nftAssetsListSelector = createSelector(nftAssetsSelector, (assets) => {
-  return assets ? List(Object.entries(assets).map((item) => item[1])) : List([])
-})
+export const nftAssetsListSelector = createSelector(
+  nftAssetsSelector,
+  (assets): List<NFTAsset> => {
+    return assets ? List(Object.values(assets)) : List()
+  },
+)
 
 export const activeNftAssetsListSelector = createSelector(
   nftAssetsListSelector,
   safeActiveAssetsSelector,
-  (assets, activeAssetsList) => {
-    return assets.filter((asset: any) => activeAssetsList.has(asset.address))
+  (assets, activeAssetsList): List<NFTAsset> => {
+    return assets.filter(({ address }) => activeAssetsList.has(address))
   },
 )
 
-export const safeActiveSelectorMap = createSelector(activeNftAssetsListSelector, (activeAssets) => {
-  const assetsMap = {}
-  activeAssets.forEach((asset: any) => {
-    assetsMap[asset.address] = asset
-  })
-  return assetsMap
-})
+export const safeActiveSelectorMap = createSelector(
+  activeNftAssetsListSelector,
+  (activeAssets): NFTAssets => {
+    const assetsMap = {}
```

Comment:
It could be simplified using `activeAssets.reduce`

Reply:
yes, quite old and not related... but :man_shrugging: it's a tiny fix